index.html

```diff
@@ -4,180 +4,228 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script
-      src="https://kit.fontawesome.com/06aa7828ae.js"
-      crossorigin="anonymous"
-    ></script>
     <link rel="stylesheet" href="styles.css" />
     <title>Portfolio</title>
   </head>
   <body>
     <header class="header">
-      <div class="mobile-nav">
-        <a href="#" class="toolbar logo"><img src="images/logo.png" alt="" /></a>
-        <label>&#9776;</label>
-      </div>
-      <div class="desk-nav">
-        <a href="#">Portfolio</a>
-        <a href="#">About</a>
-        <a href="#">Contact</a>
-      </div>
-    </header>
-    <main class="main-sec">
```

Comment:
- Please just as the previous reviewer requested changes on the mobile menu, please implement the changes. 
  - On the mobile menu when the user clicks on the hamburger menu it shows the menu, but there is no `X` to close the menu. 
  - The menu design does not look like that of the Figma design, please fix it so it looks similar to the Figma design. 

##### Your design
![Screenshot from 2022-06-15 12-37-01](https://user-images.githubusercontent.com/58771507/173818984-4cf713b6-673e-4030-a9b8-b55b7e469d38.png)

##### Figma design
![Screenshot from 2022-06-15 12-43-19](https://user-images.githubusercontent.com/58771507/173819366-1bb4ecc2-02c7-43f3-adf7-cc4cc928c49c.png)